public/stylesheets/carousel.css

```diff
@@ -0,0 +1,123 @@
+
+
+* {
+	box-sizing: border-box;
```

Comment:
Consider using spaces instead of tabs - this ensures that anyone opening your code in their editors will see the same thing that you do (tab stops can vary, spaces do not).